config.js

```diff
@@ -34,6 +34,12 @@ module.exports = {
   // See `bufferSize`.
   flushDelaySeconds: 30,
 
+  // If paths are present in this array, then these paths will be ignored before
+  // `samplingRate` based decisions are made. Paths must include a leading
+  // forward slash and be of the form:
+  //   /componentOne/componentTwo/...
+  ingoreUrls: [],
```

Comment:
Done. We now accept both regex and direct string matches. I don't want people to have to escape '/' as direct matches will likely be the common case and paths have lots of them.